rbac/internal/views.py

```diff
@@ -480,7 +480,7 @@ def get_param_list(request, param_name):
 def data_migration(request):
     """View method for running migrations from V1 to V2 spiceDB schema.
 
-    POST /_private/api/utils/data_migration/?exclude_apps=cost_management,rbac&orgs=id_1,id_2&write_db=True
+    POST /_private/api/utils/data_migration/?exclude_apps=cost_management,rbac&orgs=id_1,id_2&write_relationships=True
```

Comment:
I renamed this parameter throughout to make it a little clearer.

Now that migration involves writes to the local DB (BindingMapping), "write_db" was confusing to me and seemed to refer to the _local_ DB, even though what this really means is whether relation tuples will be written out directly (or just persisted in outbox).

So hopefully this name is clearer but happy to revert or try something else.